task_spec/Boltdir/Puppetfile

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+forge "http://forge.puppetlabs.com"
+
+mod 'puppetlabs-facts',
```

Comment:
I ran in to multiple problems trying to support the `repositories` key in `.fixtures.yml` for Travis CI and Jenkins CI. I found that just installing facts with bolt and then using fixtures.yml to build a symlink to be a simpler solution.

Reply:
I'm concerned we're still using fixtures.yaml to "install" the puppet_agent module for testing but a pupeptfile for the facts module. I think we need to manage all code with a single tool

Reply:
I think that as soon as we release fact and do not need to point to the `donoghuc` fork of the repository within fixtures.yml we can use only `fixtures.yml` instead of `Puppetfile`. I just tested in Jenkins and it works there. 

When the commit is merged to puppetlabs-facts I will see if I can clone the repo in Travis from puppetlabs fork. 

The message for Travis is: 
```
2.14s$ cd task_spec && bundle exec rake task_acceptance
Cloning into 'spec/fixtures/modules/facts'...
Warning: Permanently added the RSA host key for IP address '192.30.253.112' to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.
rake aborted!
Failed to clone git repository git@github.com:donoghuc/puppetlabs-facts.git into spec/fixtures/modules/facts
/home/travis/.rvm/gems/ruby-2.3.1/gems/puppetlabs_spec_helper-2.11.0/lib/puppetlabs_spec_helper/tasks/fixtures.rb:136:in `clone_repo'
/home/travis/.rvm/gems/ruby-2.3.1/gems/puppetlabs_spec_helper-2.11.0/lib/puppetlabs_spec_helper/tasks/fixtures.rb:300:in `block (3 levels) in <top (required)>'
Tasks: TOP => task_acceptance => spec_prep
(See full trace by running task with --trace)
```